app/controllers/employees_controller.rb

```diff
@@ -16,13 +16,13 @@ def new
   def create
       @employee = Employee.new(employee_params) 
       respond_to do |format| 
-      if @employee.save 
-        format.html { redirect_to employees_url, notice: 'Employee was successfully created.' } 
-      else 
-        format.html { render :new } 
-      end 
+        if @employee.save
+          format.html { redirect_to employees_url, notice: 'Employee was successfully created.' }
+        else
+          format.html { render :new }
+        end
+      end
```

Comment:
Is a good practice squashing these kinds of commits with the original we've received feedback. Of course, just if is a small change like this one 😄 

For example, I'll squash this commit with this one https://github.com/Brightcoders-Bootcamps/kata-07-el-reloj-checador-TM/pull/32/commits/274a5d54c060a71aee6349064da70b6f9c5bf69e

For squashing commits, you can follow these steps:
1. Start an interactive rebase with `master`
```bash
git fetch && git rebase origin/master --interactive
```
2. Once you get the list of commits that are going to be rebased, just pick the ones you want to squash just below the ones are going to be melded, and replace `pick` with `squash`. You'd found more details about how does this work just below the list of commits or you can also refer to the [documentation](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) 
For example, for this specific scenario:

| Before | After |
| - | - | 
| ![image](https://user-images.githubusercontent.com/39539196/100266606-b4436100-2f17-11eb-9d57-84ae7ec87885.png) | ![image](https://user-images.githubusercontent.com/39539196/100266781-fff60a80-2f17-11eb-9dac-c47a94175f25.png) |
3. Just save and close this file and the rebase will be performed

Sadly due to the multiple changes, there are along all commits, it causes multiple rebase-conflicts haha
Feel free to leave it as is, is not a big problem, or take it as an opportunity for practicing this technique of rewriting your git-history.

The only benefit of making this is getting a cleaner git-history, and therefore it looks more presentable and professional 😄 

While just fixing the indentation is trivial, it's a commit that is no really adding value to the project.

Reply:
Great suggestion right there ☝️ , just a comment, try to use `fixup` instead of `squash` for this, since you'll likely want to keep the first commit message only, but the whole explanation still applies, thanks Eduardo!